blogposts/2021/how-to-not-break-a-search-engine-or-what-i-learned-about-unglamorous-engineering.md

```diff
@@ -230,12 +235,12 @@ was finally dropped, it felt good and it felt right.
 
 While building the new parser I ran countless searches and constantly jumped
 around the codebase to figure out how things interacted. It made me realize
-that developer tools like code search is really just empowering me to do
+that developer tools like code search are really just empowering me to do
 unglamorous engineering more effectively. That's a very, very good thing. Of
 course tools like code search aren't exclusive to humdrum work, but I wonder
 how much value developers ultimately derive from tools that reduce unglamorous
 effort. I bet it's substantial. Refactoring and rewriting, searching for
-dependency versions and updating them, this _is_ the stuff of unglamorous
+dependency versions and updating them—this _is_ the stuff of unglamorous
```

Comment:
Should the emphasis be on "this" rather?

Reply:
don't feel strongly :-) taking your suggestion